### PR TITLE
Backwards compatibility for changes in 0.4

### LINKED ISSUE #33: broken on julia master
```
julia> using KNITRO
WARNING: both Docile and Base export "@doc"; uses of it in module KNITRO must be qualified
ERROR: LoadError: UndefVarError: @doc not defined
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:197
 in require at ./loading.jl:146
while loading /home/mlubin/.julia/v0.4/KNITRO/src/KNITRO.jl, in expression starting on line 31
```

### PR DESCRIPTION
**Pull request by yeesian**
fixes #33 

I don't have 0.4 running on my laptop. Can someone doublecheck that it fixes the issue?

### CODE REVIEW

**Comment by mlubin** [comment]
```
ERROR: LoadError: LoadError: expression is not a function call or symbol
 in error at ./error.jl:21
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:197
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:197
 in require at ./loading.jl:146
while loading /home/mlubin/.julia/v0.4/KNITRO/src/ktr_functions.jl, in expression starting on line 141
while loading /home/mlubin/.julia/v0.4/KNITRO/src/KNITRO.jl, in expression starting on line 333
```

**Comment by yeesian** [comment]
okay, bit the bullet

**Comment by yeesian** [comment]
Whoops forgot to test on 0.3...

File: src/ktr_params.jl [hunk 1]
```diff
@@ -12,116 +12,116 @@ const rev_var_type_map = Compat.@compat Dict(
...
-"KTR_PARAM_MIP_HEUR_MAXTIMEREAL" => int32(2025),
-"KTR_PARAM_MIP_PSEUDOINIT" => int32(2026),
-"KTR_PARAM_MIP_STRONG_MAXIT" => int32(2027),
-"KTR_PARAM_MIP_STRONG_CANDLIM" => int32(2028),
-"KTR_PARAM_MIP_STRONG_LEVEL" => int32(2029),
-"KTR_PARAM_PAR_NUMTHREADS" => int32(3001),
-"KTR_PARAM_PAR_CONCURRENT_EVALS" => int32(3002),
-"KTR_PARAM_PAR_BLASNUMTHREADS" => int32(3003),
-"KTR_PARAM_PAR_LSNUMTHREADS" => int32(3004)
+"KTR_PARAM_NEWPOINT" => Compat.@compat Int32(1001),
```
Comment (mlubin) [inline src/ktr_params.jl]: Try using parentheses with the macros, e.g., ``@compat(Int32(1001))``

Comment (yeesian) [inline src/ktr_params.jl]: that resolves it, thanks!

**Comment by mlubin** [comment]
Thanks for the fix!

**Comment by mlubin** [comment]
This still isn't working on 0.4:
```
julia> using KNITRO
ERROR: LoadError: UndefVarError: @docstrings not defined
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:197
 in require at ./loading.jl:146
while loading /home/mlubin/.julia/v0.4/KNITRO/src/KNITRO.jl, in expression starting on line 6
```

**Comment by yeesian** [comment]
whoops, I commented out that line (`@docstrings`) in my local 0.4 directory, but forgot to copy over to the 0.3 (git branch) version. Seems like it isn't needed there too. I'll open another PR just to be sure.

**Pull request merged**